libs/cosmos-sdk/baseapp/baseapp_parallel.go

```diff
@@ -78,7 +78,7 @@ func (app *BaseApp) paraLoadSender(txs [][]byte) {
 			for txBytes := range ch {
 				tx, err := app.txDecoder(txBytes)
 				if err == nil {
-					app.getTxFee(checkStateCtx.WithTxBytes(txBytes), tx)
+					app.getTxFee(checkStateCtx.WithTxBytes(txBytes), tx, true)
```

Comment:
this line may be set flase，i think this function is parallel verify signature.  so must be set fasle to verify signature.